crates/storage_models/src/payment_intent.rs

```diff
@@ -29,6 +29,7 @@ pub struct PaymentIntent {
     pub setup_future_usage: Option<storage_enums::FutureUsage>,
     pub off_session: Option<bool>,
     pub client_secret: Option<String>,
+    pub attempt_id: Option<String>,
```

Comment:
Should I remove payment attempt as optional and make it mandatory. Or is there any case where we would create a payment_intent but not have the attempt_id ?

Reply:
I made it mandatory.

Reply:
we need to made this as non optional.